example/protocol-adapter-example/scripts/create_hono_device.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+#*******************************************************************************
+# Copyright (c) 2016, 2020 Contributors to the Eclipse Foundation
```

Comment:
current year only
this probably applies to all the other files you have added as well ...